benefits/core/widgets.py

```diff
@@ -11,16 +11,17 @@ class FormControlTextInput(widgets.TextInput):
     def __init__(self, pattern=None, placeholder=None, **kwargs):
         super().__init__(**kwargs)
 
-        self.attrs.update({"class": "form-control form-control-lg"})
+        self.attrs.update({"class": "form-control"})
         if pattern:
             self.attrs.update({"pattern": pattern})
         if placeholder:
             self.attrs.update({"placeholder": placeholder})
 
 
 class RadioSelect(widgets.RadioSelect):
```

Comment:
This class could also be renamed to `VerifierRadioSelect` to help with clarity

Reply:
Renamed and also added a docstring https://github.com/cal-itp/benefits/pull/1023/commits/6ad1eadb2a57aace29e241b986fb4cf5754868a8